<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlTabWidget</class>
 <widget class="QWidget" name="ControlTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidgetControl">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="LoadPos" name="tabPos">
      <attribute name="title">
       <string>导入</string>
      </attribute>
     </widget>
     <widget class="LoadImageWgt" name="tabImage">
      <attribute name="title">
       <string>导入图相</string>
      </attribute>
     </widget>
     <widget class="ConfigParam" name="tabConfig">
      <attribute name="title">
       <string>设置参数</string>
      </attribute>
     </widget>
     <widget class="Refocus" name="tabRefocus">
      <attribute name="title">
       <string>重聚焦</string>
      </attribute>
     </widget>
     <widget class="SaveResult" name="tabSave">
      <attribute name="title">
       <string>保存结果</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoadPos</class>
   <extends>QWidget</extends>
   <header>maincontent/controlwidget/loadpos.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Refocus</class>
   <extends>QWidget</extends>
   <header>maincontent/controlwidget/refocus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigParam</class>
   <extends>QWidget</extends>
   <header>maincontent/controlwidget/configparam.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoadImageWgt</class>
   <extends>QWidget</extends>
   <header>maincontent/controlwidget/loadimagewgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SaveResult</class>
   <extends>QWidget</extends>
   <header>maincontent/controlwidget/saveresult.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidgetControl</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
